COORD/iocBoot/iocCOORD-IOC-01/st_riken_port_changeover.cmd

```diff
@@ -0,0 +1,8 @@
+epicsEnvSet(PC_OK_TO_RUN_PSUS,$(MYPVPREFIX)SIMPLE:VALUE1)
+epicsEnvSet(PC_ALLOW_CHANGEOVER,$(MYPVPREFIX)SIMPLE:VALUE2)
+epicsEnvSet(PC_PSU_DISABLE,$(MYPVPREFIX)$(IOCNAME):PC:PSUS:DISABLE)
+epicsEnvSet(PC_PSU_POWER,$(MYPVPREFIX)$(IOCNAME):PC:PSUS:POWER)
+
+dbLoadRecords("$(TOP)/db/riken_changeover.db","PV_PREFIX=$(MYPVPREFIX),P=$(MYPVPREFIX)$(IOCNAME):,OK_TO_RUN_PSUS=$(PC_OK_TO_RUN_PSUS),PSU_DISABLE=$(PC_PSU_DISABLE),PSU_POWER=$(PC_PSU_POWER)")
+dbLoadRecords("$(TOP)/db/riken_port_changeover_psus.db","PV_PREFIX=$(MYPVPREFIX),P=$(MYPVPREFIX)$(IOCNAME):,OK_TO_RUN_PSUS=$(PC_OK_TO_RUN_PSUS),PSU_DISABLE=$(PC_PSU_DISABLE),PSU_POWER=$(PC_PSU_POWER)")
+dbLoadRecords("$(TOP)/db/riken_changeover_groups.db","PV_PREFIX=$(MYPVPREFIX),P=$(MYPVPREFIX)$(IOCNAME):,OK_TO_RUN_PSUS=$(PC_OK_TO_RUN_PSUS),PSU_DISABLE=$(PC_PSU_DISABLE),PSU_POWER=$(PC_PSU_POWER)")
```

Comment:
For readability, only pass the macros which are actually required.

Reply:
In which case do we need to set them as EPICS env variables at all or can we just pass them straight to the seq in the st.cmd?

Reply:
The reason you can't pass them straight to seq in the individual st.cmds is that starting a sequence program before `iocInit` is called is a bad idea (undefined behaviour?)